src/main/kotlin/no/risc/github/GithubConnector.kt

```diff
@@ -73,6 +73,11 @@ data class Author(val name: String?, val email: String?, val date: Date) {
     fun formattedDate(): String = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'").format(date)
 }
 
+data class UpdateOrCreateHelper(
```

Comment:
Kanskje vi kan endre navnet på denne?
Haha, fortsatt vanskelig med navn, men hva med noe som;
```suggestion
data class RiScApprovalPRStatus(
```

Reply:
Enig! Har oppdatert til det nå 👍